config/logging.yml

```diff
@@ -4,6 +4,10 @@ rootLogger: ${es.logger.level}, console, file
 logger:
   # log action execution errors for easier debugging
   action: DEBUG
+
+  # deprecation logging, turn to DEBUG to see them
```

Comment:
Maybe be even more explicit that this logger only receives logs at the debug level, so any higher level will not log any message